technical/consumers-and-profiles/rio.md

```diff
@@ -241,12 +241,13 @@ Notes:
 | ------------------------------------------ | -------------------------------------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| programId  [*]                             | aangebodenOpleidingCode [1]                  |                                                              | De unieke aanduiding van een aangeboden opleiding voor uitwisseling in de keten.                                                                                      |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|                                            | aangebodenOpleidingCode [1]                  |                                                              | De unieke aanduiding van een aangeboden opleiding voor uitwisseling in de keten.                                                                                      | RIO will assign the value of `programId` to the `aangebodenOpleidingCode` in case of a newly created AangebodenHoOpleiding. However, it is not guaranteed that they will have the same value at all times and in all situations. See <code style="font-size:82%;">eigenAangebodenOpleidingSleutel</code> for mapping of `programId`.                                                                                                                                                                           |
```

Comment:
Dit is niet helemaal accuraat. Het is niet RIO die de waarde toekent, maar de RIO mapper. Althans als de RIO mapper degene is die de AangebodenOpleiding voor het eerst aanmaakt.

Reply:
voorstel:
The value of `courseId` will be assigned to the `aangebodenOpleidingCode` in case of a newly created AangebodenHoOpleiding.